### PR TITLE
Opgops 1902

### PR DESCRIPTION
**Pull request by tim-mcarthur-moj**
curator config for

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN  rm -f /etc/apt/sources.list.d/* && \
 RUN pip install -U pip && pip install awscli Beaver==36.2.0
 
 # ConfD install
-RUN  wget -O /usr/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.9.0/confd-0.9.0-linux-amd64 && \
+RUN  wget -O /usr/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.11.0/confd-0.11.0-linux-amd64 && \
```
Comment (carinadigital) [inline base/Dockerfile:33]: Is this a fully backwards compatible release?

File: elasticsearch5/docker/confd/templates/curator.config.tmpl [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  ssl_no_validate: False
+  http_auth:
+  timeout: 30
+  master_only: False
+
+logging:
+  loglevel: INFO
+  logfile: /var/log/app/curator.log
+  logformat: default
+  blacklist: ['elasticsearch', 'urllib3']
```
Comment (brettminnie) [inline elasticsearch5/docker/confd/templates/curator.config.tmpl]: new line missing

File: elasticsearch5/env [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
-ELASTICSEARCH_GATEWAY_RECOVER_AFTER_NODES=1
\ No newline at end of file
+ELASTICSEARCH_GATEWAY_RECOVER_AFTER_NODES=1
+
+#Curator settings
+ELASTICSEARCH_PRUNE_INDICES='yes'  #default 'no'
+ELASTICSEARCH_PRUNE_INDEX_0_NAME="metricbeat-"
+ELASTICSEARCH_PRUNE_INDEX_1_NAME="filebeat-"
+ELASTICSEARCH_PRUNE_INDEX_2_NAME="logstash-"
+ELASTICSEARCH_INDICES_PERSIST_WEEKS=2
```
Comment (brettminnie) [inline elasticsearch5/env]: new line missing

**Review by brettminnie — CHANGES_REQUESTED** [review]
2 New lines for pendatrys sake

**Pull request merged**